packages/cli/oclif.manifest.json

```diff
@@ -184,12 +197,12 @@
           "description": "Shop URL. It can be the shop prefix (janes-apparel) or the full myshopify.com URL (janes-apparel.myshopify.com, https://janes-apparel.myshopify.com).",
           "multiple": false
         },
-        "no-json-output": {
-          "name": "no-json-output",
+        "json-output": {
```

Comment:
Might be worth adding a changelog entry for this, too.

Reply:
I'm not sure. It is not a real change, the flag still works exactly the same, only the help command would change.